git/libgit2/README.md

```diff
@@ -0,0 +1,6 @@
+# Decommission notice
+
+The `github.com/fluxcd/pkg/git/libgit2` has been decommissioned as it is no longer used in the Flux project.
+
+The last package version that ships with libgit2 is
+[git/v0.10.0](https://github.com/fluxcd/pkg/releases/tag/git%2Fv0.10.0).
```

Comment:
we should also mention the last tagged version of the libgit2 package itself, i.e. `git/libgit2/v0.6.0`